### PR TITLE
chore(views): ensure unique textarea IDs in input/longtext

### PR DESCRIPTION
**Pull request by mrclay**

### CODE REVIEW

**Comment by hypeJunction** [comment]
LGTM

**Comment by hypeJunction** [comment]
Related #9955

File: views/default/input/longtext.php [hunk 1]
```diff
@@ -14,11 +14,17 @@
 $vars['class'] = (array) elgg_extract('class', $vars, []);
 $vars['class'][] = 'elgg-input-longtext';
 
+// unique $counter
+$config_key = 'counter:' . __FILE__;
+$counter = elgg_get_config($config_key);
+$counter += 1;
+elgg_set_config($config_key, $counter);
```
Comment (mrclay) [inline views/default/input/longtext.php]: simple counter is a bad idea. fixing...

File: views/default/input/longtext.php [hunk 2]
```diff
@@ -18,7 +18,7 @@
 	'value' => '',
 	'rows' => '10',
 	'cols' => '50',
-	'id' => 'elgg-input-' . rand(), //@todo make this more robust
+	'id' => "elgg-input-" . base_convert(mt_rand(), 10, 36),
```
Comment (mrclay) [inline views/default/input/longtext.php]: mt_rand really is fine for this. base 36 just to make it shorter.

**Pull request merged**